tests/scripts/task_python_microtvm.sh

```diff
@@ -22,8 +22,12 @@ set -x  # NOTE(areusch): Adding to diagnose flaky timeouts
 
 source tests/scripts/setup-pytest-env.sh
 
-# cleanup pycache
-find . -type f -path "*.pyc" | xargs rm -f
+# Search for __pycache__ in all checked-in dirs. As of now, no Python code is
```

Comment:
Do we want to put this in just uTVM? isn't pycache potentially a problem for us everywhere? also isn't the python interpreter in theory suppose to handle this?

Reply:
I was pretty sure there was some issue with running this locally, but I now can't remember what it was. what isn't commented, isn't. let's try running through CI at least, worst case we have exactly the logic we need to bring back in the first patch on this PR.